src/main/resources/templates/db_scripts/populateEffectiveCallNumberComponentsForExistingItems.sql

```diff
@@ -5,7 +5,8 @@ SET jsonb = jsonb_set(
   jsonb_build_object(
     'callNumber', COALESCE(it.jsonb->'itemLevelCallNumber', hr.jsonb->'callNumber'),
     'prefix', COALESCE(it.jsonb->'itemLevelCallNumberPrefix', hr.jsonb->'callNumberPrefix'),
-    'suffix', COALESCE(it.jsonb->'itemLevelCallNumberSuffix', hr.jsonb->'callNumberSuffix')
+    'suffix', COALESCE(it.jsonb->'itemLevelCallNumberSuffix', hr.jsonb->'callNumberSuffix'),
+    'typeId', COALESCE(it.jsonb->'itemLevelCallNumberTypeId', hr.jsonb->'callNumberTypeId')
```

Comment:
Can we do this here, because this script needs to be used for different versions of the module?

Reply:
It is not so clear from the documentation. I've asked @TarasSpashchenko, it seems that it is fine to extend this migration file and change  the `fromModuleVersion`.

Am I correct understand that the inventory-storage module has already been released? And new code, merged to the master, won't affect the release?

Reply:
Yes, mod-inventory-storage has been released.

I think it is potentially confusing to alter existing scripts and change when they are executed.

Thanks for asking for clarification from folks.

Reply:
https://github.com/folio-org/raml-module-builder/blob/master/DB-schema-migration.md:

It seems that the only case when the script won't be executed is when `current module version` > `fromModuleVersion`.

For example: 
current module version is: 18.2.0.
fromModuleVersion is: 17.1.0.

In this case the script/table creation/etc. won't be executed because the RMB assumes that the change has already been applied before.

But for release example:
current module version (released) is: 17.1.0.
fromModuleVersion is: 18.2.0.

The migration script should be applied.

@TarasSpashchenko, @skoczko, @julianladisch am I correct understand?

Reply:
Yes.